app/lib/coronavirus_local_restrictions_constraint.rb

```diff
@@ -0,0 +1,9 @@
+class CoronavirusLocalRestrictionsConstraint
+  def matches?(*)
+    !Rails.env.production? || redirect_from_date&.future?
```

Comment:
Minor - doesn't seem necessary to have the safe navigation operator since this will only return a time value.

Reply:
I added that in preparation in switching out the hard-coded time for config. However we won't be needing this anymore, so I'll remove the commit.